L1Trigger/TrackFindingTracklet/src/VMRouterPhiCorrTable.cc

```diff
@@ -31,7 +33,11 @@ void VMRouterPhiCorrTable::init(int layer, int bendbits, int rbits) {
   }
 
   if (settings_.writeTable()) {
-    writeVMTable("VMPhiCorrL" + std::to_string(layer_) + ".txt", false);
+    if (not std::filesystem::exists(settings_.tablePath())) {
+      system((string("mkdir -p ") + settings_.tablePath()).c_str());
```

Comment:
check if successful

Reply:
DONE
--  P.S. As this file/directory creation code appears in dozens of places, a future PR (almost ready) will move it to a single common function, to reduce duplicate code.